parts/k8s/kuberneteswinagentresourcesvmss.t

```diff
@@ -1,12 +1,15 @@
-{{if UseManagedIdentity}}
+{{if and UseManagedIdentity (not UserAssignedIDEnabled)}}
```

Comment:
ditto the new `not UserAssignedIDEnabled` condition, just making sure we want this

Reply:
same here